rdkcentral/mock-firebolt
# Add injector browser extension

- Added 'injector' browser extension
- Updated docs/readme's to refer to it

To test:
- Visit chrome://extensions
- Click Load unpacked button
- Navigate to the injector directory
- Visit a web page with a url that contains "mf="; it should connect to Mock Firebolt even if it doesn't have "activateMockFirebolt.js" in the app
